examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html

```diff
@@ -18,40 +17,46 @@
 
 <body>
 	<div class="ui-page" data-go-to-top-button="true">
-		<header class="ui-header">
-			<h2 class="ui-title">
-				Data usage
+		<header>
+			<h2>
+				Title
 			</h2>
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
 				<li class="li-has-checkbox">
 					<label>
-						Mobile Data
-						<input type="checkbox" />
-					</label>
-				</li>
-				<li class="li-has-checkbox">
-					<label>
-						Mobile Data
-						<input type="checkbox" />
-					</label>
-				</li>
-				<li class="li-has-checkbox">
-					<label>
-						Mobile Data
-						<input type="checkbox" />
+						Check on
+						<input type="checkbox" checked />
 					</label>
 				</li>
 				<li class="li-has-checkbox">
 					<label>
-						Mobile Data
+						Check off
 						<input type="checkbox" />
 					</label>
 				</li>
 			</ul>
 		</div>
 	</div>
+	<script src="../../../js/app.js">
+	</script>
+	<script src="../../../js/lowBatteryCheck.js">
+	</script>
+	<script>
```

Comment:
the problerm with not working script (about which I wrote in previous comment) may be that you are not including this inside of 
<div class="ui-page" data-go-to-top-button="true">

Reply:
Yes, the problem is that while navigation only page is taken into account, others elements are not.